### PR TITLE
Added empty 2D unity game.

### PR DESCRIPTION
**Pull request by JackDevMats**

### CODE REVIEW

**Comment by JackDevMats** [comment]
before u approve, pull the branch and see if it can be added to your unity hub

**Review by kiblitz — APPROVED** [review]
I assume nothing is supposed to happen upon clicking play.

LGTM

Feel free to merge and delete this branch

**Comment by JackDevMats** [comment]
yep, merging

**Pull request merged**
